numpy/testing/nosetester.py

```diff
@@ -326,9 +325,9 @@ def test(self, label='fast', verbose=1, extra_argv=None,
              <http://nedbatchelder.com/code/modules/coverage.html>`_).
         raise_warnings : str or sequence of warnings, optional
             This specifies which warnings to configure as 'raise' instead
-            of 'warn' during the test execution.  Valid strings are:
+            of 'always' during the test execution.  Valid strings are:
 
-              - "develop" : equals ``(DeprecationWarning, RuntimeWarning)``
+              - "develop" : equals ``(Warning,)``
```

Comment:
Yes, I like the severe interpretation ;). Though scipy might need some fixing up. One good thing is that even if you remove it, the warnings will still get printed once. On the other hand I see no problem with putting back the explicit warning checks in those tests.